example/pyo3/run.py

```diff
@@ -17,7 +17,7 @@ def main() -> None:
     )
     logger = logging.getLogger(__name__)
 
-    (acceleration_mode, open_jtalk_dict_dir, text, out) = parse_args()
+    (acceleration_mode, open_jtalk_dict_dir, text, out, SPEAKER_ID) = parse_args()
```

Comment:
pythonは`snake_case`が慣例なので、それに倣うと良さそう？
```suggestion
    (acceleration_mode, open_jtalk_dict_dir, text, out, speaker_id) = parse_args()
```